zebra-network/src/protocol/external/message.rs

```diff
@@ -401,8 +401,6 @@ impl TryFrom<Message> for VersionMessage {
     }
 }
 
-// TODO: add tests for Error conversion and Reject message serialization (#4633)
-// (Zebra does not currently send reject messages, and it ignores received reject messages.)
 impl<E> From<E> for Message
```

Comment:
This test coverage is still missing, if we ever modify this code we should add these tests:

```suggestion
// TODO: add tests for Error conversion and Reject message serialization
 // (Zebra does not currently send reject messages, and it ignores received reject messages.)
impl<E> From<E> for Message
```